### ISSUE TITLE
Should re really adn-allow everything from easylist & uBlock filters ?

### ISSUE THREAD

**Issue by mneunomne** [dhowe/AdNauseam#2149]
I wonder if our current approach of adn-allowing everything that comes from easylist & uBlock filters is the best way of making sure we collect ads.

What I notice is that we will always tend to have ads many appearing on websites that are ad-aggressive (e.g. https://ww1.theofficeepisode.com/). And doesn't matter how much effort we put in re-blocking the adn-allowed requests because the ad system are always changing.

I was taking a look in easylist filters and noticing there are various segments of the list, regarding popups, documents, generic blocks, domain-speific etc. I think we should perhaps take a closer look in these adn-allowed lists and check what do we really need adn-allow and what we don't.

**Comment by mneunomne** [comment]
For example, we should block all request that generate popups (new browser windows being generated).

https://github.com/easylist/easylist/blob/master/easylist/easylist_specific_block_popup.txt

**Comment by dhowe** [comment]
Yes, ideally there would be a way to determine such rules in our parsing of the list and then we could choose to block those...

**Comment by mneunomne** [comment]
Maybe we can add these specific .txt files to our filters and simply check? Wonder if perhaps we can ask help for the EasyList people by creating an issue there.

**Comment by dhowe** [comment]
so are these .txt files combined into the single list that adblockers then download ?

**Comment by mneunomne** [comment]
> so are these .txt files combined into the single list that adblockers then download ?

@dhowe seems like yes, they are merged into one big file.

(going back to this now, to-do's) 


- [ ] Pick a website that is ads heavy, that some ads can go through AdNauseam
- [ ] Document what types of ads are being generated (popup's?, ghost-links?, google ads?)
- [ ] Manually add different easylist filters from [this lists](https://github.com/easylist/easylist/tree/master/easylist) and see if the ads disappear while still collecting.
